app/mailers/user_notification_mailer.rb

```diff
@@ -0,0 +1,31 @@
+class UserNotificationMailer < ApplicationMailer
+  default from: 'no-reply@example.com'
+
+  def login_notification(user)
+    @greeting = "Hi"
+    @user=user 
+
+    mail(to: @user.email, subject: 'Successfully Logged in')
+
```

Comment:
bhai, you can use robocop for fixing indentation, but don't run it for entire file, run it only where it is necessary